### PR TITLE
Fix request authentication safe hash comparison

### PR DESCRIPTION
**Pull request by sstok**
|Q            |A  |
|---          |---|
|Bug Fix?     |yes|
|New Feature? |no |
|BC Breaks?   |no  |
|Deprecations?|no |
|Tests Pass?  |yes|
|Fixed Tickets|   |
|License      |MIT|

### CODE REVIEW

**Comment by Ener-Getick** [comment]
:-1: this class [is deprecated](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/Security/Core/Util/StringUtils.php#L14) in favor of ``hash_equals``.

**Comment by sstok** [comment]
@Ener-Getick fixed 👍

File: src/AppBundle/Issues/GitHubRequestHandler.php [hunk 1]
```diff
@@ -10,6 +10,7 @@
 use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
 use Symfony\Component\HttpKernel\Exception\PreconditionFailedHttpException;
 use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+use Symfony\Component\Security\Core\Util\StringUtils;
```
Comment (WouterJ) [inline src/AppBundle/Issues/GitHubRequestHandler.php]: can be removed

Comment (sstok) [inline src/AppBundle/Issues/GitHubRequestHandler.php]: done

**Comment by Ener-Getick** [comment]
:+1: fine for me now ☺

**Comment by xabbuh** [comment]
You may want to require the Symfony Polyfill package now to be able to still use this on older PHP versions.

**Comment by WouterJ** [comment]
As carson is a symfony app, it should already be included when needed

**Comment by weaverryan** [comment]
Thanks Sebastiaan.

**Pull request merged**